deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -346,7 +396,12 @@ Resources:
                 - aws s3 rm s3://$DEST_BUCKET --recursive
             build:
               commands:
-                - aws s3 cp --recursive . s3://$DEST_BUCKET/
+                - SOURCE_SHA=$(cat sha.txt)
+                - aws s3 cp --recursive . s3://$DEST_BUCKET/ --exclude "sha.txt"
+                - DATE=$(date)
+            post_build:
+              commands:
+                - curl -i -X POST -H "Authorization:token ${OAUTH_TOKEN}" "${ApiURL}/${SOURCE_SHA}" -d "{\"state\":\"success\",\"description\":\"Deployed to production on ${DATE}\",\"context\":\"${CODEBUILD_INITIATOR}\",\"target_url\":\"${TargetURL} "}"
```

Comment:
I think this is missing an escape in the cURL command, and should probably read: 

`- curl -i -X POST -H "Authorization:token ${OAUTH_TOKEN}" "${ApiURL}/${SOURCE_SHA}" -d "{\"state\":\"success\",\"description\":\"Deployed to production on ${DATE}\",\"context\":\"${CODEBUILD_INITIATOR}\",\"target_url\":\"${TargetURL}\"}"`

Reply:
Fixed missing escape.